portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,142 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class ="nav">
+        <nav>
+        <ul>
+     <a href="#aboutme">About Me</a>
+    <a href="#projects">Projects</a>
+     <a href="#myprofile" >Experience</a>
+    <a href="#contact" >Contact Me</a>
+     <a href="#translate">Translate</a>
+    </ul>
+</nav>
     </div>
+    <div class="first">
+        <h1>Hi, I am <span>Kun</span> Chen</h1>
+    </div>
+    <div id="kc">
+        <h2><center>KUN CHEN</center></h2>
+        <p>734-496-3088</p>
+        <p>kunchenkc7@gmail.com | Ann Arbor, MI</p>
+        <p>U of M Class of 2024</p>
+        <p>Bachelor of Science in Computer Science</p>
+        <p>Bachelor of Arts in Economics</p>
+    </div>
+      <div id="aboutme">
+    <h2><center>ABOUT ME</center></h2>
+    <a href="photo.html">
+    <img src="Image/2.jpg" 
+    width="300"
+    height="400"
+    align="center" />
```

Comment:
Make sure your tags are properly tabbed up. I recommend using a formatter online to format your code automatically.

https://webformatter.com/html